index.html

```diff
@@ -0,0 +1,357 @@
+<!DOCTYPE html>
+<html lang="nl">
+<head>
+    <meta charset="UTF-8">
+    <meta content="IE=edge" http-equiv="X-UA-Compatible">
+    <meta content="width=device-width, initial-scale=1.0" name="viewport">
+    <meta content="detailpagina funda" name="description">
+
+    <!-- Link to CSS -->
+    <link href="styles/style.css" rel="stylesheet" type="text/css">
+
+    <!-- Link to boxicons -->
+    <link href='https://unpkg.com/boxicons@2.1.4/css/boxicons.min.css' rel='stylesheet'>
+
+
+    <!-- Link to JavaScript script -->
+
+    <script defer src="scripts/v2.js">
+
+    </script>
+    <!--geen defer vanwege error in w3c validator-->
+
+    <title>Detailpagina</title>
+    <!--    hier werkt de rode knp weer niet oorzaak niet bekend-->
+</head>
+<body><!--https://dequeuniversity.com/screenreaders/-->
+<header>
+    <a href="https://www.funda.nl/koop/assendelft/huis-42349645-pronkgevel-1/">
+        <img alt="logo funda" class="log-funda" src="images/fundawonen-logo.svg">
+    </a>
+    <!--       <i class='bx bx-menu hamburger'></i>-->
+    <button class="hamburger-icon" type="button"><img alt="menu icon" class="" src="images/menubar.png"></button>
+    <nav class="menu">
+
+        <ul class="menu-ul">
+            <li class="menu__li carousel-images__no-dot"><a href="#" class="menu-link">kopen</a></li>
+            <li class="menu__li carousel-images__no-dot"><a href="#" class="menu-link">huren</a></li>
+            <li class="menu__li carousel-images__no-dot"><a href="#" class="menu-link">verkopen</a></li>
+            <li class="menu__li carousel-images__no-dot"><a href="#" class="menu-link">Mijn huis</a></li>
+            <li class="menu__li carousel-images__no-dot"><img alt="login icon" src="images/log-in-regular-24.png"></li>
+
+        </ul>
+
+    </nav>
+</header>
+
```

Comment:
I think you can add a single return consistently instead of differences between 1 return, 2 returns or 2 returns. Add them consistently for each element.